charts/gardener-extension-shoot-lakom-service/values.yaml

```diff
@@ -59,3 +59,8 @@ controllers:
   debugConfig:
     enableProfiling: false
     enableContentionProfiling: false
+
+gardener:
+  seed:
+    annotations:
+      service.lakom.extensions.gardener.cloud/deploySeedResources: true
```

Comment:
Is it necessary to set these helm values here? They will be considered as default values when the chart is installed. 

 
Also, the annotation key should not be camelCase, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions

> Key names should be all lowercase, with words separated by dashes instead of camelCase

For annotation key I would recommend `seed.lakom.extensions.gardener.cloud/enable-admission-controller`

Reply:
Wouldn't we want them exactly for using them as default values when the annotation is not added?

Reply:
> For annotation key I would recommend seed.lakom.extensions.gardener.cloud/enable-admission-controller
My idea with the naming was to make it clear that the controller is still enabled but it just won't deploy the resources monitoring and validating the images on the seed.

Does `seed.lakom.extensions.gardener.cloud/enable-admission-controller` make the same thing clear? I don't want someone to interpret the option as completely disabling the controller.

Reply:
> Not for values that gardener manages. The values files are also used as reference document what options are available and we should not expose what gardener is additionally injecting.
Thanks, I hadn't thought about this.

> but it can be improved a bit more if the annotation key is named seed.lakom.extensions.gardener.cloud/enable-lakom-admission-controller
I agree, I'll use this.

Reply:
Actually, I'll hold out on making the change for now because of the discussion about deciding whether we're on a managedseed based on the existance of a `validatingwebhookconfiguration`.